src/pretix/control/templates/pretixcontrol/subevents/detail.html

```diff
@@ -145,11 +145,13 @@ <h4 class="panel-title">
                         </div>
                         <div class="form-group subevent-itemvar-group">
                             <div class="col-md-4 col-md-offset-3">
-                                <label for="{{ f.available_from.id_for_label }}" class="text-muted">{% trans "Available from" %}</label><br>
-                                {% bootstrap_field f.available_from form_group_class="" layout="inline" %}
+                                <label for="{{ f.available_from.id_for_label }}" class="text-muted">{% trans "Available from" %}</label>
+                                {% include "pretixcontrol/subevents/fragment_unavail_mode_indicator.html" with mode=f.available_from_mode %}<br>
+                                {% bootstrap_field f.available_from form_group_class="foo" layout="inline" %}
                             </div>
                             <div class="col-md-4">
-                                <label for="{{ f.available_until.id_for_label }}" class="text-muted">{% trans "Available until" %}</label><br>
+                                <label for="{{ f.available_until.id_for_label }}" class="text-muted">{% trans "Available until" %}</label>
+                                {% include "pretixcontrol/subevents/fragment_unavail_mode_indicator.html" with mode=f.available_until_mode %}<br>
```

Comment:
This should probably be added to `bulk_edit.html` and `bulk_add.html` as well

Reply:
Done ✅